api/python/gdh.py

```diff
@@ -0,0 +1,101 @@
+# Depends: requests pandas
+# You may already have these installed, otherwise install using pip:
+#     pip install requests pandas
+
+# Example invocations:
+# >>> from gdh import cases, qs
+# >>> c1 = cases(qs(country="New Zealand"), "API KEY HERE")
+# >>> c2 = cases(qs(country="New Zealand", admin1="Auckland"), "API KEY HERE")
+#
+# You can use the cases_cached() function to cache the cases locally.
+# This is useful for rerunning the script which will then use the
+# cached version.
+
+import os
+import io
+import json
+import hashlib
+import datetime
+
+import requests
+import pandas as pd
+
+GDH_URL_DEFAULT = "https://data.covid-19.global.health"
+GDH_URL = os.getenv("GDH_URL", GDH_URL_DEFAULT)
+downloadAsync = "/api/cases/downloadAsync"
+
+# sync with data-serving/data-service/src/util/search.ts
+FILTERS = {
+    "curator",
+    "gender",
+    "nationality",
+    "occupation",
+    "country",
+    "outcome",
+    "caseid",
+    "uploadid",
+    "sourceid",
+    "sourceurl",
+    "verificationstatus",
+    "admin1",
+    "admin2",
+    "admin3",
+    "variant",
+    "dateconfirmedafter",
+    "dateconfirmedbefore",
+}
+
+
+def qs(**kwargs):
```

Comment:
Does this function turn filters into a string? If so, maybe a better name would be something like `stringify_filters`?